lib/DataSourceFactory.php

```diff
@@ -164,7 +164,7 @@ private function checkDataSourceName($name, DataSourceInterface $datasource = nu
             throw new DataSourceException('Name of data source must be unique.');
         }
 
-        if (!preg_match('/^[\w\d]+$/', $name)) {
+        if (!preg_match('/^[\w]+$/', $name)) {
```

Comment:
`if (1 !== preg_match('/^[\w]+$/', $name)) {`?

Reply:
srsly?

Reply:
Yeah you're doing minor cs fixes anyway.

Reply:
but not everywhere